README.md

```diff
@@ -114,9 +114,11 @@ impl World for HelloServer {
 Lastly let's write our `main` that will start the server. While this example uses an
 [in-process
 channel](https://docs.rs/tarpc/0.18.0/tarpc/transport/channel/struct.UnboundedChannel.html),
-tarpc also ships a
-[transport](https://docs.rs/tarpc-bincode-transport/0.7.0/tarpc_bincode_transport/)
-that uses bincode over TCP.
+tarpc also ships
+[bincode](https://docs.rs/tarpc-bincode-transport/0.7.0/tarpc_bincode_transport/)
+and
+[JSON](https://docs.rs/tarpc-json-transport/0.1.0/tarpc_json_transport)
+transports that use respective formats over TCP.
```

Comment:
It might be worth mentioning that they're both generic over anything that reads and writes bytes, but they also provide tokio-net based TCP functionality.